conf/test_full.config

```diff
@@ -1,138 +1,25 @@
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_POS_Lib/P1rC_POS_180522171703.mzML',
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_POS_Lib/P1rD_POS_180522175927.mzML',
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_POS_Lib/P1rB_POS_180522163438.mzML',
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_POS_Lib/P1rA_POS_180522155214.mzML'
-  ]
-
-  id_library_mzml_files_pos = [
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_POS_Lib/P1rC_POS_180522171703.mzML',
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_POS_Lib/P1rD_POS_180522175927.mzML',
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_POS_Lib/P1rB_POS_180522163438.mzML',
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_POS_Lib/P1rA_POS_180522155214.mzML'
-  ]
-
-  library_description_pos = 'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/library_charac_pos.csv' // Path to a csv file containing description of the library for positive (see the help)
-
-  quant_library_mzml_files_neg = [
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_NEG_Lib/P1rC_NEG.mzML',
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_NEG_Lib/P1rD_NEG.mzML',
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_NEG_Lib/P1rB_NEG.mzML',
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_NEG_Lib/P1rA_NEG.mzML'
-  ]
-
-  id_library_mzml_files_neg = [
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_NEG_Lib/P1rC_NEG.mzML',
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_NEG_Lib/P1rD_NEG.mzML',
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_NEG_Lib/P1rB_NEG.mzML',
-    'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/mzML_NEG_Lib/P1rA_NEG.mzML'
-  ]
-
-  library_description_neg = 'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/library_charac_neg.csv' // Path to a csv file containing description of the library for negative (see the help)
-
-  // MetFrag
-  database_csv_files_pos_metfrag = 'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/hmdb_2017-07-23.csv' // path to a csv file containing your database
+params {
+    config_profile_name        = 'Full test profile'
+    config_profile_description = 'Full test dataset to check pipeline function'
 
-  // CFM-ID
-  database_csv_files_pos_cfmid = 'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/hmdb_2017-07-23.csv' // path to a csv file containing your database
+    // Input data for full size test
+    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/metaboigniter/samplesheet.csv'
```

Comment:
same sample than for the normal test?

Reply:
No additional samples will be used in the full test.